contracts/BaseStrategy.sol

```diff
@@ -306,85 +266,55 @@ abstract contract BaseStrategy {
     }
 
     constructor(address _vault) public {
-        _initialize(_vault, msg.sender, msg.sender, msg.sender);
+        _initialize(_vault, msg.sender, msg.sender, msg.sender, 300);
     }
 
-    /**
-     * @notice
-     *  Initializes the Strategy, this is called only once, when the
-     *  contract is deployed.
-     * @dev `_vault` should implement `VaultAPI`.
-     * @param _vault The address of the Vault responsible for this Strategy.
-     * @param _strategist The address to assign as `strategist`.
-     * The strategist is able to change the reward address
-     * @param _rewards  The address to use for pulling rewards.
-     * @param _keeper The adddress of the _keeper. _keeper
-     * can harvest and tend a strategy.
-     */
     function _initialize(
         address _vault,
         address _strategist,
         address _rewards,
-        address _keeper
+        address _keeper,
+        uint256 _changeLimitRatio
```

Comment:
prefer to add a sensible default so we don't break existing strat code and is easy to just extend from this one on existing strat code